docs/setting-up/client/index.md

```diff
@@ -331,29 +333,47 @@ dnf localinstall *.rpm
 
 5. Run the installer.
 
+    !!! caution alert alert-warning "Non root users"
+        Run this command before next step:
+        ```sh
+        export PMM_DIR=YOURPATH
+        ```
+        where YOURPATH replace with you real path, where you have required access.  
+
+        **Node, MySQL and PostgreSQL exporters wont be avalaible.**
+
+    This command require root permissions (if you skipped step for non root users).
```

Comment:
should this end up with `:` because it is not super clear which step requires root. Or maybe shift it to it's own enum, `6.`

Reply:
@denisok Changed to same style like in others places. Check out screenshot here: https://jira.percona.com/secure/attachment/28755/Screenshot%20from%202021-07-07%2012-53-17.png